### ISSUE TITLE
Log out issue

### ISSUE THREAD

**Issue by birdofpreyru** [topcoder-platform/community-app#36]
Now when `Log Out` is triggered via top nav menu (at the moment, we have two top nav menus in this repo: one for the main web-site, another for the TC Community mini-sites), it just redirects user to `https://www.topcoder-dev.com/logout`. This redirection does log out the user from Topcoder `accounts-app`, but it does not wipe out auth cookies associated with the `community-app` subdomain. It is not that obvious, because when the App is restarted it communicates with `accounts-app`, and cleans the cookies if the user is not authenticated, but there is a short moment before this communication, when the user may see something he is not supposed to see.

In other words, the best way to see this issue:
1. Login as a valid user and go to Wipro community page.
2. Log out through the top nav menu.
3. Press `back` in the browser until you return to the Wipro community page: you will see the Wipro community page will be visible for a brief time, and only a moment later it will change to the `You must be authenticated to access this page` message.
